### ISSUE TITLE
Experiencing issue where autofill list is centered and cut off

### ISSUE THREAD

**Issue by funkadunkle** [turing753/minimal-functional-fox#4]
Title, here is a photo for reference:
![image](https://user-images.githubusercontent.com/29989582/70981920-efcc7e80-206a-11ea-94c7-9a67e23a07f8.png)

I've tried changing text alignment for the urlbar list but have had no luck as of yet.
Would greatly appreciate any help.

**Comment by mut-ex** [comment]
Have you considered moving all those icons to the right of the URL bar into the overflow menu? They will still be just a click away but that will reduce the clutter and prevent the URL bar from being pushed all the way to the left.

If you don't want to do that, look for this (on line 252)

`.urlbarView-row {
	text-align: center !important;
}`

And change center to left.

Moreover on line 238 you will find this selector: `.urlbarView:not(.megabar)`. You can override, and play with the `margin-left`, and `width` properties to adjust the list as per your liking.

**Comment by funkadunkle** [comment]
I moved all the icons to overflow, and I had already changed that center to left, but with no fix. I currently have disabled the `width` and `margin-left` properties, it certainly doesn't look as pretty, but it makes it so that even with the indented text, it is still readable at least.

It seems the error only occurs the first time going to the URL bar, every time after that, the text is aligned as intended. Strange.
